wallet/src/de/schildbach/wallet/ui/CreateUsernameActivity.kt

```diff
@@ -78,44 +85,100 @@ class CreateUsernameActivity : InteractionAwareActivity(), TextWatcher {
             showCompleteState()
         }
 
+        walletApplication = application as WalletApplication
```

Comment:
Can we move the two observers below, or their setup, outside the onCreated method?

Reply:
yes

Reply:
observers moved to initViewModel method here:  07f06f1